planning/behavior_path_planner/src/planner_manager.cpp

```diff
@@ -342,6 +342,34 @@ std::vector<SceneModulePtr> PlannerManager::getRequestModules(
   return request_modules;
 }
 
+BehaviorModuleOutput PlannerManager::getReferencePath(
```

Comment:
Should this function be removed ?

Reply:
In the series of this task, I have moved member function implementations from header to source files if the function is long enough to have very few expectations of getting inlined, OR the function requires boost.geometry calculation /lanelet2_extension functions because they introduce a lot of symbols into the header as well as the propagated source files.

Reply:
Sorry I did not know about that. Thank you for the explanation and for the great PRs to improve the includes :bow: